.circleci/config.yml

```diff
@@ -15,22 +15,28 @@ jobs:
           name: Build
           command: |
             make all -j4
+            make tracer -j4
             make test -j4
       - run:
           name: Run unit tests
           command: |
             export PYTHONPATH=$SMAUG_HOME/build:$PYTHONPATH
             make test-run
+      - run:
+          name: Run simulation sweeps
+          command: |
+            # Download the gem5 binary.
+            python .circleci/download_artifacts.py --api_token=${GEM5_ALADDIN_BUILD_ARTIFACT_TOKEN} --project=gem5-aladdin --artifact_name=gem5.opt --user=${USER} --download_loc=/tmp --filter=${BUILD_ARTIFACT_FILTER} --branch=${BUILD_ARTIFACT_BRANCH}
+            chmod +x /tmp/gem5.opt
+            ln -s /usr/lib/x86_64-linux-gnu/libtcmalloc_minimal.so.4.3.0 /usr/lib/x86_64-linux-gnu/libtcmalloc.so.4
```

Comment:
Hm, why is this necessary? I thought we use (basically) the same Docker image to build gem5-aladdin?

Reply:
Found the culprit. I didn't install `libgoogle-perftools-dev` in the SMAUG docker image, which caused the issue. Added a commit for that.